t/Renard/Curie/Component/ScrollZoom.t

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env perl
+
+use Test::Most tests => 4;
+
+use lib 't/lib';
+use Renard::Curie::Setup;
+use CurieTestHelper;
+use Renard::Curie::Types qw(InstanceOf Enum);
+
+my $cairo_doc = CurieTestHelper->create_cairo_document;
+
+fun Scroll_Event( (InstanceOf['Renard::Curie::App']) $app,
+		(Enum[qw(up down smooth)]) $direction,
+		$delta = 0) {
```

Comment:
@helenbach I made a change on these two lines so that it is clearer about what is expected for those parameters.

The `Enum` type restricts the `$direction` variable to the kinds that are accepted for the scroll event.

The `= 0` gives it a default value.